src/compare_component_and_website_data/tmp.py

```diff
@@ -0,0 +1,36 @@
+"""Compare"""
+def clean(ls):
+    """Reads lines and cleans out common issues"""
+    return set([
+        x.replace('\\u00a0', '')
+        .replace('\xa0', '')
+        .replace('\n', '')
+        for x in ls])
+
+
+# Read files
+# - Component OWL
+with open('comp2.txt', 'r') as f:
+    c_lines = f.readlines()  # 5,781
+    c_set = clean(c_lines)  # 2,331
+# - Alignment product1.json
+with open('json2.txt', 'r') as f:
+    j_lines = f.readlines()  # 5,781
+    j_set = clean(j_lines)  # 2,331
+# Nomenclature Excel / XML
+with open('nomenclature.txt', 'r') as f:
+    n_lines = f.readlines()  # 4,101
+    n_set = clean(n_lines)  # 1,951
+
+# Analysis
+# - OWL and JSON set diff: 0
+source_diff1 = c_lines != j_lines  # True
+i1 = c_set.intersection(j_set)  # 2,332 100%
+c1_diff = c_set.difference(j_set)  # 0
+j_diff = j_set.difference(c_set)  # 0
+
+# - OWL and nomenclature set diff:
```

Comment:
## Analysis: Difference of ORDO OWL to nomenclature file(s)
[Nico asked me](https://github.com/monarch-initiative/icd11/pull/12#pullrequestreview-1965582809) to compare our / the Orphanet OWL to the nomenclature file(s)--namely, the set difference/similarity of ICD11MMS references.

I started again by manually creating a file of all of the refs, `nomenclature.txt`. I did this by copying from the "ICD-11 MMS codes" column of `ORPHAnomenclature_MasterFile_2023.xlsx`. I then used the same code as my other analysis.

Results: